src/Controllers/LoggerController.js

```diff
@@ -4,6 +4,8 @@ import AdaptableController from './AdaptableController';
 import { LoggerAdapter } from '../Adapters/Logger/LoggerAdapter';
 
 const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
+const LOG_STRING_TRUCATE_LENGTH = 1000;
```

Comment:
Can I say no? ;).  Seems like getting back to the way logs were on parse is a good starting point?  You wanna do a commit for it?  Otherwise, I'll get to it, but would rather not hold this up for that, not a lot of work, I assume, but I'll have to learn how to do, document and unit test it.  

Thanks for catching the typo!  doh.